apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md

```diff
@@ -10,7 +10,6 @@ keywords:
     Permissionless output proposals,
     Permissionless challenges,
     Withdrawals,
-    Base Sepolia testnet,
```

Comment:
should we add Mainnet or is that covered with `Base` above?

Reply:
c3ebf304